modules/dynamic-plugins/ref-rh-supported-plugins.template.adoc

```diff
@@ -0,0 +1,19 @@
+// This page is generated! Do not edit the .adoc file, but instead run rhdh-supported-plugins.sh to regen this page from the latest plugin metadata.
+// cd /path/to/rhdh-documentation; ./modules/dynamic-plugins/rhdh-supported-plugins.sh; ./build/scripts/build.sh; google-chrome titles-generated/main/plugin-rhdh/index.html
+
+== {company-name} and partner plugins
+
+{company-name} and its partners support the following %%COUNT_1%% plugins:
+
+[%header,cols=4*]
+|===
+|*Name* |*Plugin* |*Version* |*Path and required variables*
+%%TABLE_CONTENT_1%%
+|===
+
+[NOTE]
+====
+* For more information about configuring Keycloak, see xref:rhdh-keycloak_{context}[Installation and configuration of Keycloak].
+
+* For more information about configuring TechDocs, see Configuring the TechDocs plugin in {product}.
+====
```

Comment:
I'd suggest moving this block of text to the `assembly-rhdh-installing-dynamic-plugins.adoc` file, immediately after the include statement for this file i.e. line 25.

This will allow this content to be updated without having to re-run the tables-builder script.

Reply:
Sure, works for me.